File: Test/test_cases/Test1.java
import java.io.*;
class Test
{
  public int test() {
    Seller x1=new Seller(100);
    if(x1.getQuantity()==100)
    {
      x1.setQuantity(20);
      if(x1.getQuantity()==20)
      {
        return 1;
      }
      else
      {
        return 0;
      }
    }
    else
    {
      return 0;
    }
  }
}

File: Test/test_cases/python3/tests/Test1.py
from Seller import Seller

class Test:
	def test(self):
		seller = Seller(100)
		if seller.getQuantity()==100:
			seller.setQuantity(20)
			if seller.getQuantity()==20:
				return 1
			else:
				return 0
		else:
			return 0
